packages/main/src/plugin/experimental-feature-feedback-form.spec.ts

```
@@ -0,0 +1,224 @@
+  get: configurationGetMock,
+  has: () => true,
+  update: () => Promise.resolve(),
+};
+
+class TestExperimentalFeatureFeedbackForm extends ExperimentalFeatureFeedbackForm {
+  override timestamps: Map<string, Timestamp> = new Map<string, Timestamp>();
+  override disabled: string[] = [];
+  override experimentalFeatures: Set<string> = new Set<string>();
+
+  override setTimestamp(feature: string, days: Timestamp): void {
+    return super.setTimestamp(feature, days);
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)